ext/npm-convert.js

```diff
@@ -122,6 +122,10 @@ function convertPropertyNamesAndValues (context, pkg, map, root, waiting) {
 		if(typeof name !== 'undefined' && typeof val !== 'undefined') {
 			clone[name] = val;
 		}
+		// keep map entry if the key isn't a package but value might
+		if (name && typeof val === "undefined") {
```

Comment:
@matthewp is this a naive "package" detection? I couldn't think of a different way of "knowing" the key/val is a package without a full npm name which we won't have in this case.

Reply:
Yeah, I think this is fine.